src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Clearbooks\Labs\Toggle\Entity;
 
-interface ActivatableToggle
+interface ActivatableToggle extends Toggle
```

Comment:
But LABS-2 should have information about users,groups and releases
@Clearbooks-Tom